include/flatbuffers/idl.h

```diff
@@ -455,6 +455,17 @@ struct ServiceDef : public Definition {
   SymbolTable<RPCCall> calls;
 };
 
+struct FullyQualifiedType {
```

Comment:
What is the use of this wrapper? If it was only useful during refactoring, please remove again.

Reply:
The idea was to enforce beginning with `::` as an invariant of the type, rather than having hidden requirements on a `std::string` variable.

Reply:
There's a LOT of strings thruout FlatBuffers that have expectations of the format contained in them (e.g. paths). I don't feel this case is special enough to warrant encoding the invariant in a class.